templates/jdk11/Dockerfile

```diff
@@ -41,6 +41,7 @@ RUN apt-get update && \
 	apt-get install --no-install-recommends --yes jattach && \
 	apt-get upgrade --yes && \
 	apt-get clean && \
+	echo "${TARGETARCH} ${LABEL_ZULU_11_VERSION}" && \
```

Comment:
@Wykenz, why did you need to set this?

Reply:
oh  sorry it just stayed there by mistake, I only used that line for testing and I will remove it